### ISSUE TITLE
MSCW: Clearer Product Management Communication

### ISSUE THREAD

**Issue by sonora** [osmandapp/OsmAnd#17167]
@vshcherb 
Suggestion to Product Management which could be of value: We have
- old issues which have stayed open with label "_Nice to Have_" for sometimes many years. There is no telling if we have any intention to come back to these, if their implementation by whoever wants would be appreciated, or if implementation is unwanted and they are just clutter in our issues list
- issues where "_Nice to Have_" seems an ambiguous or inappropriate comment from a user/functional point of view, where e.g. something may truly be an objective enhancement of app functionality, if however perhaps only for certain use cases

I wonder if Product Management would consider adopting the "**MSCW**" scheme to tag things more explicitly, this would communicate and document much more clarity of what each decision actually is:
- **(M) Must Have:**
_Essential requirements that are critical to the success of the project. They are the minimum set of requirements. Any requirement that falls into this category is non-negotiable._
- **(S) Should Have:**
_Requirements are important but not essential. They are not critical to the success of the project but would enhance the product's usability and value. These requirements are less critical than the Must-haves but still add value to the  product._
- **(C) Could Have:**
_Requirements are desirable but not necessary. They are nice-to-have features that can be delivered if time. These requirements have a lower priority than the Must-haves and Should-haves._
- **(W) Won’t Have:**
_Requirements are low priority or unnecessary. They can be deferred or dropped from the product backlog if they are deemed unnecessary or if there is not enough time or resources to deliver them._

**Comment by tradavyvy** [comment]
I do not know how you internaly use the list with issues, but
Maybe it would be an idea to clean out the older issues (the oldest of the 2500 open ones are from 2015).

Best case: someone from the team goes through them and closes all the ones that stem from before version 4.0 that have no or low priority.

Least effort: just close all the ones that stem from before version 4.0

In both cases leave an comment before closing, explaining that this is a cleanup action. Explain that if the 'partissipants' of this issue still think this issue is still relevant for version 4.4.x and up, they can post a new issue describing their (updated) request or bug-report (with a refference to the clossed issue number). Github sents an email to partissipants of an issue so they will be notified.

My guess is that manny issues may no longer be 'current' and the open-list will be much shorter and more relevant.
I think most people who have an old issue that is still relevant for them will not mind restating and or updating it.
It might also help clear duplicates.

Keep up the good work.

**Comment by sonora** [comment]
Yeah,good points. Done the cleanup several tines over the years. Just thought there may be a better way to stay aligned with priorities and decisions, a little documented transparency here  is for everone's benefit, I guess.
